### DISCUSSION TITLE
Build docker images for aarch64

### DISCUSSION THREAD

**Discussion by jzbor** [monkeytypegame/monkeytype#5367] [Ideas] [answered]
Hi, I wanted to self-host monkeytype on my aarch64 server, but sadly the images do not seem to be available for aarch64. This should not be too difficult, given the images are built via Github actions anyway.

**Comment by fehmer** [comment]
i'll have a look if we can build for `linux/arm64`

**Comment by fehmer** [comment, marked as answer]
hi @jzbor,

i cannot test the images myself, can you give it a try?

use the docker-compose.yml from the SELF_HOSTING documentation and replace
`monkeytype/monkeytype-frontend:latest` with `fehmer/mt-frontend:latest` and  `monkeytype/monkeytype-backend:latest` with `fehmer/mt-backend:latest`.

Thanks.

**Reply by jzbor** [reply]
Thanks a lot. I can confirm that your image works!
